src/main/resources/db/test/migration/V900__case_note_types_data.sql

```diff
@@ -82,20 +82,20 @@ VALUES ('b5aa1318-e874-4614-a80d-a514e84d1a56', 'L6962XX', 'WNI', 'ADAM_GEN', 'J
 INSERT INTO OFFENDER_CASE_NOTE_AMENDMENT (OFFENDER_CASE_NOTE_AMENDMENT_ID, OFFENDER_CASE_NOTE_ID, AUTHOR_USERNAME,
                                           AUTHOR_NAME, NOTE_TEXT, CREATE_DATE_TIME, CREATE_USER_ID, MODIFY_DATE_TIME,
                                           MODIFY_USER_ID, AUTHOR_USER_ID, SOFT_DELETED)
-VALUES (21, 'b5aa1318-e874-4614-a80d-a514e84d1a54','ADAM_GEN','John Smith','Note amendment1','2023-12-29 17:08:46.655233','ADAM_GEN','2023-07-29 17:08:46.655233',
+VALUES (-21, 'b5aa1318-e874-4614-a80d-a514e84d1a54','ADAM_GEN','John Smith','Note amendment1','2023-12-29 17:08:46.655233','ADAM_GEN','2023-07-29 17:08:46.655233',
```

Comment:
Likely my inability to read the int tests properly. I can't figure out why these now have negative ids!